dev/content/notebooks/Random Variable Algebra.ipynb

```diff
@@ -0,0 +1,597 @@
+{
```

Comment:
<p>I don't know if this is GitHub not rendering right, but here I'm seeing the same image as above again, "Expectation algebra for random variables". Perhaps "Variance algebra for random variables" should go here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/canyon289/canyon289.github.io/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>nvm, looks like it was indeed GitHub not rendering correctly, if I open this with nbviewer it looks correct https://nbviewer.jupyter.org/github/canyon289/canyon289.github.io/blob/rv_alg/dev/content/notebooks/Random%20Variable%20Algebra.ipynb</p>